src/dotnet/commands/dotnet-migrate/DotnetNewRedirector.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Cli;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Tools.MigrateCommand;
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.Tools.Migrate
+{
+    public class DotnetNewRedirector : ICanCreateDotnetCoreTemplate
+    {
+        public void CreateWithWithEphemeralHiveAndNoRestore(
```

Comment:
And a typo in the worst place possible... in the interface let me change me update the cli-migrate